mysql-test/CMakeLists.txt

```diff
@@ -55,109 +57,108 @@ INSTALL(
       RENAME README-test
       )
   ENDIF()
-ENDIF()
-
 
 
-IF(NOT ${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_BINARY_DIR})
-  # Enable running mtr from build directory
-  CONFIGURE_FILE(
-    ${CMAKE_CURRENT_SOURCE_DIR}/mtr.out-of-source 
-    ${CMAKE_CURRENT_BINARY_DIR}/mysql-test-run.pl
-    @ONLY
-  )
-ENDIF()
-IF(UNIX)
-  EXECUTE_PROCESS(
-    COMMAND chmod +x  mysql-test-run.pl
-    COMMAND ${CMAKE_COMMAND} -E create_symlink 
-     ./mysql-test-run.pl mtr
-    COMMAND ${CMAKE_COMMAND} -E create_symlink 
-     ./mysql-test-run.pl mysql-test-run
-    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-  ) 
-  IF(INSTALL_MYSQLTESTDIR)
-    INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/mtr 
-      ${CMAKE_CURRENT_BINARY_DIR}/mysql-test-run 
-      DESTINATION ${INSTALL_MYSQLTESTDIR}
-      COMPONENT Test)
+  IF(NOT ${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_BINARY_DIR})
+    # Enable running mtr from build directory
+    CONFIGURE_FILE(
+      ${CMAKE_CURRENT_SOURCE_DIR}/mtr.out-of-source 
+      ${CMAKE_CURRENT_BINARY_DIR}/mysql-test-run.pl
+      @ONLY
+    )
+  ENDIF()
+  IF(UNIX)
+    EXECUTE_PROCESS(
+      COMMAND chmod +x  mysql-test-run.pl
+      COMMAND ${CMAKE_COMMAND} -E create_symlink 
+       ./mysql-test-run.pl mtr
+      COMMAND ${CMAKE_COMMAND} -E create_symlink 
```

Comment:
this entire change in file is only because of the new IF block?

Reply:
@satya-bodapati yes, I've adjusted formatting, so the extra tab caused all this. But the only change is https://github.com/percona/percona-xtrabackup/pull/1093/files/33539bb93f966798637676150098f7569f529efa..368a7de038823cc15b89e4736164a4499d0358bf#diff-25b9cb7fd9753fffb13a4219e2f67eb22373660c6aaf7126eaa0a8442001ec6eR25 and https://github.com/percona/percona-xtrabackup/pull/1093/files/33539bb93f966798637676150098f7569f529efa..368a7de038823cc15b89e4736164a4499d0358bf#diff-25b9cb7fd9753fffb13a4219e2f67eb22373660c6aaf7126eaa0a8442001ec6eR164